Feat/auth bypass

techlabs-be-devgit opened the pull request:
This branch implements programmatic login for the C2C web application.

jasjaap-brl committed:
Add auth.js and authsettings.js

jasjaap-brl committed:
Add code to login by interacting with Entra ID API endpoint

Commit:
Delete cypress/support/authsettings.json

jasjaap-brl committed:
corrected variable name in auth.js

jasjaap-brl committed:
Implement programmatic login to C2C web application

jasjaap-brl committed:
1. Add working programmatic login() functionality for the C2C application.
2. Remove secrets from the cypress.config.js file.
3. Write basic test in spec.cy.js and remove dummy code.
4. Update README to include dependencies for the automation.
5. Add code to ignore uncaught application exceptions in e2e.js

jasjaap-brl committed:
Edit readme

jasjaap-brl committed:
Rename spec.cy.js to clientManagement.cy.js

luigibayrocklabs reviewed (changes requested):
tiny changes requested, then were good to go

jasjaap-brl committed:
1. Track node_modules
2. remove extra blank line from cyprress/e2s/clientManagement.cy.js
3. Change README.md
